ccd_snr/sections/_model.py

```diff
@@ -0,0 +1,329 @@
+Where $P_m < 1$ (in the \UV\ and visible) the Fano factor,
+as measured from the front face of the sensor,
+will be much larger than unity since fewer photons are detected in this region.
+"""
+    )
+    subsection_noise.append(subsubsection_noise_shot)
+    subsubsection_noise_fano = aastex.Subsubsection("Fano Noise")
+    subsubsection_noise_fano.append(
+        r"""
+The energy resolution of silicon detectors is ultimately limited due to Fano
+noise \citep{Fano1947}, the unpredictable variation of \QY.
+Fano noise is usually expressed in terms of a Fano factor, 
+$\mathcal{F} = \sigma^2 / \mu$,
+the ratio of the variance to the mean for some random process.
+Expressing noise in this fashion is convenient because for a Poisson random
+sampling, $\mathcal{F} = 1$ (which is signal-independent, unlike the \SNR).
+
+The Fano noise for silicon is commonly accepted to have a Fano factor of about 
+$\mathcal{F} \approx 0.1$ \citep{Janesick2001}.
+In part due to variations of the Fano noise as a function of wavelength and
+temperature \citep{Fraser1994}, 
+there is some disagreement in the literature around a more precise value for
+$\mathcal{F}$ 
+\citep[\& references therein]{Fraser1994,Lowe1997,Mazziotta2008,Kotov2018,Rodrigues2021,Rodrigues2023}.
+$\mathcal{F}$ is often measured in the soft X-ray region,
+traditionally with $^{55}$Fe sources, which have a high \QY.
+For \UV\ wavelengths, where the \QY\ is near unity, it becomes impossible
+to construct a distribution narrow enough to be consistent with a Fano factor 
+that small.
+Because this distribution does not exist,
+and because $\mathcal{F}$ is so small compared to the other noise sources
+considered in this study, 
+we have decided to ignore the wavelength variation of $\mathcal{F}$,
+and adopt a Fano noise model with constant $\mathcal{F} = \fanoFactor$,
+which represents the best available measurement of $\mathcal{F}$ using $^{55}$Fe
+X-rays \citep{Rodrigues2021}, and uses a skipper CCD \citep{Janesick1990} to 
+minimize the effect of readout noise.
+
+At high energies, the \PDF\ of the Fano noise is well-described by a Gaussian
+\citep{Rodrigues2023}.
+At low energies, a Gaussian model is problematic since it becomes likely
+that $\text{IQY}(\lambda)$ will be negative for some samples.
+For this work, we will use a scaled Poisson distribution,
+\begin{equation}
+    P(\text{QY}=k) = \frac{[\text{IQY}(\lambda) / \mathcal{F}]^{\mathcal{F} k} e^{-\text{IQY}(\lambda) / \mathcal{F}}}
+                          {(\mathcal{F} k )!},
+\end{equation}
+which has the nice property of reproducing a Gaussian with the correct width
+at high energies while also being well-behaved around
+$\text{IQY}(\lambda) \approx 1$.
+In Figures \ref{fig:photonNoise} and \ref{fig:electronNoise} we can see the 
+contribution of Fano noise to the total noise measured by our simulated sensor.
+Note how the Fano noise component is very small compared to the photon shot noise.
```

Comment:
The thing we have to get right is the expectation value (references), so in the low IQY regime the variance must increase in order to make this possible (when the IQY is not an integer).

Also, well-behaved is not very specifice, say how do it does not go negative.